goosebit/api/devices/models.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import time
+from enum import StrEnum
+from typing import Annotated
+
+from pydantic import BaseModel, BeforeValidator, computed_field
+
+from goosebit.models import Device, UpdateModeEnum, UpdateStateEnum
+from goosebit.updater.manager import get_update_manager
+
+
+class UpdateStateModel(StrEnum):
```

Comment:
I would like to figure out how to dynamically update these enums so that they always match up with the corresponding database version,  ideally these only every have to be updated on one place.  Any ideas here LMK, I dont know a way off the top of my head but im sure its possible.

Reply:
I think I got this in 2a7d4a79f944a10d93c36054000783778f5db56a.